packages/main/src/themes/sap_horizon/Panel-parameters.css

```diff
@@ -1,12 +1,12 @@
 @import "../base/Panel-parameters.css";
 
 :root {
-	--_ui5_panel_header_height: 2.75rem;
 	--_ui5_panel_button_root_width: 2.75rem;
-	--_ui5_panel_background_color: var(--sapGroup_ContentBackground);
 	--_ui5_panel_border_radius: var(--sapElement_BorderCornerRadius);
 	--_ui5_panel_border_bottom: none;
+	--_ui5_panel_default_header_border: 0.0625rem solid var(--sapGroup_TitleBorderColor);
 	--_ui5_panel_outline_offset: -0.125rem;
 	--_ui5_panel_title_font_weight: bold;
```

Comment:
Bold title font-weight wasn't present in the spec, looks closer to the target design without it.